use openshift-gitops operator  in sec-hub managed cluster

jeffdmgit opened the pull request:
Perhaps since the sec-hub cluster was not provisioned by the hub but rather previously running then imported by the hub/rhacm, then the openshift-gitops operator needs to be added explicitly.

nickmerrett commented:
can you remove the openshift-gitops-operator stuff as we dont need that now, its done through policy

jeffdmgit commented:
> can you remove the openshift-gitops-operator stuff as we dont need that now, its done through policy

Removed `2-services/argocd/instances/openshift-gitops-instance.yaml` and `2-services/argocd/operators/openshift-gitops.yaml` and removed *- argocd/instances/openshift-gitops-instance.yaml* from `0-bootstrap/spokeclusters/infra/sec-hub/2-services/kustomization.yaml`.

I also commented out the ibm-platform-navigator operator and instance from `0-bootstrap/spokeclusters/infra/sec-hub/2-services/kustomization.yaml`, which I used months ago as a workaround for an issue where they don't get launched/deployed. I never got confirmation from the cntk team back then on whether this really was necessary or not.

nickmerrett reviewed (approved):
ok looks fine
